digiwf-task/pom.xml

```
@@ -18,7 +18,7 @@
     <properties>
         <kotlin.version>1.9.22</kotlin.version>
         <camunda.version>7.20.0</camunda.version>
-        <polyflow.version>4.1.0</polyflow.version>
+        <polyflow.version>4.1.1-SNAPSHOT</polyflow.version>

```

Wieso die SNAPSHOT-Version? Die verhindert gerade den Build.
```suggestion
        <polyflow.version>4.1.1</polyflow.version>

```